### PR TITLE
ost ims access token parsing

### PR DESCRIPTION
**Pull request by Axelcureno**
- Sets adobeid object with IMS client details.
- Fetches IMS access token to parse when initializing OST in Milo when user is authenticated.
- Signs into IMS when user is not authenticated and redirects back to OST.

Resolves: [MWPW-122375]([MWPW-122375](https://jira.corp.adobe.com/browse/MWPW-122375))

**Test URLs:**
- Before: https://ost--milo--adobecom.hlx.page/tools/ost
- After: [https://mwpw-122375--milo--adobecom.hlx.page/tools/ost](https://mwpw-122375--milo--adobecom.hlx.page/tools/ost)

### CODE REVIEW

File: libs/blocks/ost/ost.js
Comment (npeltier) [inline libs/blocks/ost/ost.js:12]: no need for adobeio scope?

File: libs/blocks/ost/ost.js
Comment (npeltier) [inline libs/blocks/ost/ost.js:27]: i would default to PROD here

File: libs/blocks/ost/ost.js
Comment (npeltier) [inline libs/blocks/ost/ost.js:29]: and here to some prod user

File: libs/blocks/ost/ost.js
Comment (yesil) [inline libs/blocks/ost/ost.js:6]: since we load OST styles externally, we can remove blocks/ost/ost.css

File: tools/sidekick/config.json
Comment (vladen) [inline tools/sidekick/config.json:75]: 👍 1

File: libs/blocks/ost/ost.js
Comment (vladen) [inline libs/blocks/ost/ost.js:5]: 👍

File: libs/blocks/ost/ost.js
Comment (Axelcureno) [reply libs/blocks/ost/ost.js:7]: Removed

File: libs/blocks/ost/ost.js
Comment (Axelcureno) [reply libs/blocks/ost/ost.js:13]: Is the exact value of the scope `adobeio`? I only see `adobeio_api`, `adobeio.appregistry.read` and `adobeio.appregistry.read` in IMS portal. All 3 options require business justification and approval, will wait for confirmation before submitting.
@yesil @honstar

File: libs/blocks/ost/ost.js
Comment (Axelcureno) [reply libs/blocks/ost/ost.js]: Done

File: libs/blocks/ost/ost.js
Comment (Axelcureno) [reply libs/blocks/ost/ost.js]: Done

File: libs/blocks/ost/ost.js
Comment (honstar) [reply libs/blocks/ost/ost.js:13]: I'd say omit the `adobeio` scope if it is not needed for OST to function properly. I can't imagine why `adobeio` would be needed here in the first place, sounds totally unrelated.

**Review by honstar — APPROVED** [review]

**Pull request merged**
